test/unit/parse.cpp

```diff
@@ -26,6 +26,23 @@ struct parse_test
             BOOST_TEST_NO_THROW(ru);
             BOOST_TEST(ru->query() == "~");
         }
+        // parse docs
+        {
+            result< url_view > ru = parse_relative_ref( "//www.boost.org/index.html?field=value#downloads" );
+            if ( ru.has_value() )
+            {
+                url_view u = *ru;
+                assert(u.encoded_path() == "/index.html");
+            }
+        }
+        {
+            result< url_view > ru = parse_uri_reference( "https://www.example.com/path/to/file.txt" );
+            if ( ru.has_value() )
+            {
+                url_view u = *ru;
+                assert(u.encoded_path() == "/path/to/file.txt");
```

Comment:
This could just be `rc->encoded_path()` and lose the local variable (and scope). Shorter examples are better.

Reply:
This is exemplifying `result::operator*` in the exposition.